packages/blade/src/components/Box/BaseBox/BaseBox.stories.internal.tsx

```diff
@@ -0,0 +1,152 @@
+// @TODO: test story. Remove later
+import type { Meta } from '@storybook/react';
+import type { BaseBoxProps } from './types';
+import { BaseBox } from '.';
+import StoryPageWrapper from '~src/_helpers/storybook/StoryPageWrapper';
+import { Text } from '~components/Typography';
+import { Sandbox } from '~src/_helpers/storybook/Sandbox';
+
+const baseBoxStoryArgs = {
+  padding: [
+    'Can be',
+    '- an absolute value like `"10px"`',
+    '- token `"spacing.5"`',
+    '- an array shorthand `["spacing.2", "10px", "spacing.5", "spacing.9"]',
+    '- responsive object with combinatation of all previous values `{ base: "spacing.3", l: ["spacing.10", "spacing.5"]}`',
+  ].join('\n\n'),
+  margin: [
+    'Can be',
+    '- an absolute value like `"10px"`',
+    '- token `"spacing.5"`',
+    '- an array shorthand `["spacing.2", "10px", "spacing.5", "spacing.9"]',
+    '- responsive object with combinatation of all previous values `{ base: "spacing.3", l: ["spacing.10", "spacing.5"]}`',
+  ].join('\n\n'),
+
+  backgroundColor:
+    '<i>Only available in BaseBox</i>\n\n Value can be a color token, absolute values or responsive object with token and absolute values',
+  display:
+    'CSS `display` property. Can be  \n\n - string like `block`, `flex`, `none`, etc\n\n- Responsive object `{ base: "block", m: "none" }`',
+  overflow: '',
+  flex: '',
+  flexWrap: '',
+  flexDirection: '',
+  flexGrow: '',
+  flexShrink: '',
+  flexBasis: '',
+  alignItems: '',
+  alignContent: '',
+  alignSelf: '',
+  justifyItems: '',
+  justifyContent: '',
+  justifySelf: '',
+  order: '',
+  position: '',
+  zIndex: '',
+  grid: '',
+  gridColumn: '',
+  gridRow: '',
+  gridRowStart: '',
+  gridRowEnd: '',
+  gridArea: '',
+  gridAutoFlow: '',
+  gridAutoRows: '',
+  gridAutoColumns: '',
+  gridTemplate: '',
+  gridTemplateAreas: '',
+  gridTemplateColumns: '',
+  gridTemplateRows: '',
```

Comment:
Are we supporting grid properties in Box & as part of the responsive layout features? 

If so, how will this work in react-native? considering react-native doesn't have support for grid

Reply:
These should be web only

Reply:
yup. They won't work on react-native.

Reply:
Are these also skipped as auto suggestions by TS for native files?

Reply:
might have to figure out how to configure vscode to read native types instead of web. But yes if we could do that, then it will skip